README.md

```diff
@@ -55,6 +55,16 @@ And here are links of other **HugeGraph** component/repositories:
 3. [hugegraph-commons](https://github.com/apache/incubator-hugegraph-commons) (**common & rpc** module)
 4. [hugegraph-website](https://github.com/apache/incubator-hugegraph-doc) (**doc & website** code)
 
+## Contact Us
+- Github Issues: Feedback on usage issues and functional requirements (priority)
+
+
+- Feedback Email: dev@hugegraph.apache.org
```

Comment:
Hi, @1216063060, you can refer this PR https://github.com/apache/incubator-hugegraph-toolchain/pull/462, pay attention to adding blank lines between title and content,  remove extra blank lines in contents.

Reply:
OK，3Q~